backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/cardlist/stack/BpkStackCardList.kt

```diff
@@ -0,0 +1,58 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.cardlist.stack
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import net.skyscanner.backpack.compose.cardlist.stack.internal.BpkStackCardListImpl
+import net.skyscanner.backpack.compose.icon.BpkIcon
+import net.skyscanner.backpack.compose.sectionheader.BpkSectionHeaderButton
+
+sealed class BpkStackCardAccessoryStyle {
+    data class Expand(
+        val collapsedCount: Int,
+        val expandedCount: Int,
+        val expandText: String,
+        val collapsedText: String,
+    ) : BpkStackCardAccessoryStyle()
+    data class Button(val title: String, val icon: BpkIcon, val onClick: () -> Unit) : BpkStackCardAccessoryStyle()
+}
+
+@Composable
+fun BpkStackCardList(
+    title: String,
+    description: String,
+    totalCount: Int,
+    modifier: Modifier = Modifier,
+    accessoryStyle: BpkStackCardAccessoryStyle? = null,
+    headerButton: BpkSectionHeaderButton? = null,
+    accessibilityHeaderTagEnabled: Boolean? = true,
+    content: @Composable ((Int) -> Unit),
```

Comment:
The API is different than what Figma contains, we supposed to have one API `BpkCardList` and  both Stack and Rail passed as a `layout`
any reasons behind this differences?

Reply:
The reason is the way the Rail variant was implemented. They decided to change the implementation to be 2 different implementations.
https://github.com/Skyscanner/backpack-android/pull/1986
Also I cannot change them back to align with the API table now as it will cause a breaking change.